index.html

```diff
@@ -74,8 +74,8 @@ <h1 class="title"><abbr title="Voting Information Project">VIP</abbr> Format Spe
           <h3>Naming convention</h3>
           <p>The file containing the VIP feed should be named vipFeed-[<a href="//www.itl.nist.gov/fipspubs/co-codes/states.htm">FIPS code</a> for state or county]-[election year]-[election month]-[election day].xml. If the file is zipped, the file extensions of .zip or .xml.zip are also acceptable. <p/><p/>For instance, vipFeed-19-2012-11-06.zip denotes Iowa's (the FIPS code for IA is 19) feed for the Nov 6, 2012 election.</p>
 
-          <h3>Comma-delimited flat files</h3>
-          <p>If your office is producing flat files instead of an XML file, the files must be comma-delimited .txt files. While the format of the files differs slightly from the XML, the attributes included must conform to the field specifications below.<p/><p/> Examples of the flat files are below each XML example, along with the files included to link those elements when the files are compiled into an XML feed. To read more about the flat files, see complete file header templates and look at example files, you can do so in this <a href="https://github.com/votinginfoproject/csv-templates">GitHub repository.</a> </p>
+          <h3>Comma-delimited flat files (CSV)</h3>
+          <p>If your office is producing flat files, they must be comma-delimited .txt files. The CSV column names differ slightly from XML tag names.<p/><p/> Examples of the flat files are below each XML example and include the necessary files to link those elements together as needed. To read more about the flat files, view complete file header templates, and review example files, you can do so in this <a href="https://github.com/votinginfoproject/csv-templates">GitHub repository.</a> </p>

```

Comment:
@nomadaisy Random aside (and this might actually be a question for @tie-rack), but does the filename have to end in .txt? Could it also be .csv? This is more about our intake process and how hardened it is. Since there really is no CSV standard, I can see arguing either way (i.e. either that we accept both or mandate only one).